tests/tensorflow/nightly/tf-dcn-criteo.libsonnet

```diff
@@ -94,6 +94,18 @@ local tpus = import 'templates/tpus.libsonnet';
     },
   },
 
+  local v2_32 = tpu_common {
```

Comment:
What is this change for?

Reply:
I wanted to keep this for config reference for eg., what bottom_mlp and embedding_dim would work on v2_32.